dependencies.list

```diff
@@ -1,4 +1,4 @@
 VERSION=2.4.3
-REALM_CORE_VERSION=2.3.1
-REALM_SYNC_VERSION=1.1.0
+REALM_CORE_VERSION=2.3.2
+REALM_SYNC_VERSION=1.3.0
```

Comment:
Are there any other bug fixes in these two components that we should call out in our change log?